apps/admin/backend/src/store.ts

```diff
@@ -2137,6 +2288,31 @@ export class Store {
     return rows.map((r) => r.id);
   }
 
+  /**
+   * Gets the write-in IDs for a given cast vote record's contest.
+   */
+  getCvrContestWriteInIds({
```

Comment:
This might be too specific of a query, as I could make `cvrId` and `contestId` optional. I just went with this for now since it was my use case, but I am open to advice. Similar